assets/component-rte.css

```diff
@@ -32,7 +32,9 @@
   margin-bottom: var(--media-shadow-vertical-offset);
 }
 
-.rte ul {
+.rte ul,
+.rte ol {
```

Comment:
I feel like maybe some of this will need to be moved into the `base.css` file 🤔 
As this is going to be useful not only for rte content coming from the admin but all our `richtext` settings everywhere 🤔